code-challenges.test.js

```diff
@@ -14,47 +16,116 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+//The test should expect the array to be the same length as number on input and follow the given sequence. Should be red first then green after
+
+describe('fibonacci', () => {
+    
+    it('returns an array with the numbers in fibonacci sequence', () => {
+        expect(fibonacci(6)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(fibonacci(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
 
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
-
 // b) Create the function that makes the test pass.
 
+// make array length be the number of input. I did not need to do this but i originally thought i would. i just set the loop to num.
+//number 1 and number 2 must = number 3
+//take 3rd number and make it equal to itself minus 2 and minus 1 to get number 3.
+//loop thru and make sure to go length of num
+
+function fibonacci(num) {
+    let array = [];
+    array[0] = 1;
+    array[1] = 1;
```

Comment:
yeah easy refactor into 
```javascript
    let array = [1,1]
```

Reply:
Got it. its funny because i see why it works but it doesn't feel natural to me to write it like that